### DISCUSSION TITLE
Extending prisma to support more databases

### DISCUSSION THREAD

**Discussion by FrostyTheSouthernSnowman** [prisma/prisma#15355] [Q&A] [answered]
I would like to know if there is a way to add new databases to prisma. In particular, I would like to know if there is a way to use SurrealDB with prisma. Would I have to build a package from scratch that resembled prisma or does prisma have an extension system? Thanks.

--- edit
related issue: #15346

**Comment by janpio** [comment, marked as answer]
There is currently no way to contribute support for new databases. Sorry.

**Reply by FrostyTheSouthernSnowman** [reply]
Ok. Would the pisma team be interested in implementing some sort of module or extension system?

**Reply by janpio** [reply]
Generally yes, but not right now. We do not have the capacity to prioritize this (as we currently do not think it is the most important thing to spend our time on for Prisma ORM. Watch our releases for what we do instead.).

**Reply by FrostyTheSouthernSnowman** [reply]
OK, thanks
